docs/reference-docs/dynamic-fork-task.md

```diff
@@ -21,16 +22,16 @@ There are three things that are needed to configure a `FORK_JOIN_DYNAMIC` task.
 
 1. A list of tasks or sub-workflows that needs to be forked and run in parallel.
 2. A list of inputs to each of these forked tasks or sub-workflows
-3. A task prior to the `FORK_JOIN_DYNAMIC` tasks outputs 1 and 2 above that can be wired in as in input to
+3. A task prior to the `FORK_JOIN_DYNAMIC` tasks outputs 1 and 2 above that can be wired in as input to
    the `FORK_JOIN_DYNAMIC` tasks
 
 ## Use Cases
 
-A `FORK_JOIN_DYNAMIC` is useful, when a set of tasks or sub-workflows needs to be executed and the number of tasks or
+A `FORK_JOIN_DYNAMIC` is useful when a set of tasks or sub-workflows needs to be executed and the number of tasks or
 sub-workflows are determined at run time. E.g. Let's say we have a task that resizes an image, and we need to create a
 workflow that will resize an image into multiple sizes. In this case, a task can be created prior to
 the `FORK_JOIN_DYNAMIC` task that will prepare the input that needs to be passed into the `FORK_JOIN_DYNAMIC` task. The
-single image resize task does one job. The `FORK_JOIN_DYNAMIC` and the following `JOIN` will manage the multiple
+single image resizes task does one job. The `FORK_JOIN_DYNAMIC` and the following `JOIN` will manage the multiple
```

Comment:
I think this should be singular.  the task does a resize on just one image.  single image resize.